psi4/src/psi4/libmints/angularmomentum.cc

```diff
@@ -26,242 +26,108 @@
  * @END LICENSE
  */
 
-#include "psi4/libciomr/libciomr.h"
 #include "psi4/libmints/angularmomentum.h"
 #include "psi4/libmints/basisset.h"
-#include "psi4/libmints/molecule.h"
 #include "psi4/libmints/integral.h"
+#include "mcmurchiedavidson.h"
```

Comment:
```suggestion
#include "psi4/libmints/mcmurchiedavidson.h"
```
for consistency. or feel free to leave it as is, and we'll wait and see if the full addresses have actually been necessary. :-)